pkgs/by-name/op/openpgl/package.nix

```
@@ -31,6 +31,10 @@ stdenv.mkDerivation (finalAttrs: {
     "-DTBB_ROOT=${tbb.out}"
   ];
 
+  preBuild = lib.optionals stdenv.isAarch64 ''
+    NIX_CFLAGS_COMPILE="-flax-vector-conversions $NIX_CFLAGS_COMPILE"
+  '';
```

Does the following work?

```suggestion
  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isAarch64 "-flax-vector-conversions";
```

---

I think so. I'll implement the change